src/main/java/backend/Logic.java

```diff
@@ -312,6 +338,26 @@ private boolean handleIssueLabelsUpdateResult(boolean isUpdateSuccessful,
     }
 
     /**
+     * Handles the result of updating an issue's milestone on server.
+     * Locally reverts back to the original milestone if the server update fails.
+     * @param isUpdateSuccessful
+     * @param originalIssue
+     * @return true if the server update is successful
+     */
+    private boolean handleIssueMilestoneUpdateOnServerResult(boolean isUpdateSuccessful,
+                                                          Optional<TurboIssue> originalIssue) {
+        if (!originalIssue.isPresent()) {
+            logger.error("Unable to replace issue milestone locally");
+            return false;
+        }
```

Comment:
I think it is a programmer error. This will be handled in another PR (handle an exception thrown when the original issue cannot be found in the first place).